Author: Tollef Fog Heen <tfheen@debian.org>
Description: Patch to prevent browser from caching pages (Closes: #281284)
Bug-Debian: #281284
Forwarded: no

Index: mailman2-python3/Mailman/htmlformat.py
===================================================================
--- mailman2-python3.orig/Mailman/htmlformat.py
+++ mailman2-python3/Mailman/htmlformat.py
@@ -306,7 +306,8 @@ class Document(Container):
         charset = 'us-ascii'
         if self.language and Utils.IsLanguage(self.language):
             charset = Utils.GetCharSet(self.language)
-        output = ['Content-Type: text/html; charset=%s\n' % charset]
+        output = ['Content-Type: text/html; charset=%s' % charset]
+        output.append('Cache-control: no-cache\n')
         if not self.suppress_head:
             kws.setdefault('bgcolor', self.bgcolor)
             tab = ' ' * indent
